### ISSUE TITLE
FP in 'Passwords And Secrets In Infrastructure Code' query for Common

### ISSUE THREAD

**Issue by t3mi** [Checkmarx/kics#3442]
### Platform
Terraform

### Provider
Azure

### Description
Following example produces false positive for location attribute for sure and probably for password as well as scanning just a module file with default values set doesn't raise the same issues.

```hcl
# File infra/main.tf
variable "location" {
  default = "East US 2"
}

variable "admin_account" {
  default = "adm1n"
}

variable "admin_password" {
  default = ""
}

module "test" {
    source = "../modules/sql"

    location       = var.location
    admin_account  = var.admin_account
    admin_password = var.admin_password
}

# File modules/sql/main.tf
variable "location" {
  default = "east us"
}

variable "admin_account" {
  default = "root"
}

variable "admin_password" {
  default = "pa$$"
}

resource "azurerm_sql_server" "test" {
  name                         = "test-sql"
  resource_group_name          = "rgname"
  location                     = var.location
  version                      = "12.0"
  administrator_login          = var.admin_account
  administrator_login_password = var.admin_password
}

resource "azurerm_sql_active_directory_administrator" "test" {
  server_name         = azurerm_sql_server.test.name
  resource_group_name = azurerm_resource_group.test.name
  login               = "testlogin"
  tenant_id           = "ee591259-1c6a-4108-a3ed-7726f6c925d9"
  object_id           = "98cb95ec-38a2-4f92-ae03-6aef806dc481"
}

```

**Comment by t3mi** [comment]
@rogeriopeixotocx please reopen as its not fixed.

**Comment by rogeriopeixotocx** [comment]
@t3mi sure, can you give us some details?

**Comment by t3mi** [comment]
Yes, you can grab the files content from the initial post and scan those with nightly build. False positive is triggered on `location` attribute while should not.
```diff
Passwords And Secrets In Infrastructure Code, Severity: HIGH, Results: 2
Description: Query to find passwords and secrets in infrastructure code.
Platform: Common

        [1]: infra/main.tf:16

                015:
+               016:     location       = var.location
                017:     admin_account  = var.admin_account
```
